lib/generators/templates/models/account.rb

```diff
@@ -0,0 +1,14 @@
+class Account < BookingsyncPortal::Account
+  synced id_key: :uid, local_attributes: [:email]
+
+  # Uncomment to add Rails logger
+  # def api
+  #   @api ||= BookingSync::Engine::APIClient.new(token.token, logger: Rails.logger, account: self)
+  # end
+
+  # Uncomment to add Rails logger
+  # # Used to synchronize accounts using `Account.synchronize(remove: true)`
```

Comment:
Just checked after all updates and everything works smooth for accounts with updated_since.
(Canceled accounts are getting deleted from bsa-booking)